torch/cuda/_utils.py

```diff
@@ -98,11 +137,17 @@ def check_nvrtc(result: int) -> None:
     # Get compute capability if not provided
     if compute_capability is None:
         props = torch.cuda.get_device_properties(torch.cuda.current_device())
-        compute_capability = f"{props.major}{props.minor}"
+        if not torch.version.hip:
```

Comment:
can we reverse the negation logic? `if torch.version.hip` - should be slightly easier to read